ui/packages/catalog-ui-search/src/main/webapp/component/list-create/list-create.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) Connexta, LLC
+ *
+ * <p>Unlimited Government Rights (FAR Subpart 27.4) Government right to use, disclose, reproduce,
+ * prepare derivative works, distribute copies to the public, and perform and display publicly, in
+ * any manner and for any purpose, and to have or permit others to do so.
+ */
+import React from 'react'
+import styled from 'catalog-ui-search/src/main/webapp/react-component/styles/styled-components'
+import MarionetteRegionContainer from '../../react-component/container/marionette-region-container'
+
+var ListEditorView = require('../list-editor/list-editor.view.js')
+var _ = require('lodash')
+var List = require('../../js/model/List.js')
+var store = require('../../js/store.js')
+var ConfirmationView = require('../confirmation/confirmation.view.js')
+var Marionette = require('marionette')
+
+const CreateContainer = styled.div`
+  padding: ${props => props.theme.minimumSpacing};
+`
+
+const Button = styled.button`
+  width: 100%;
+  padding: 0px ${props => props.theme.minimumSpacing};
+`
+
+class ListCreate extends React.Component {
+  constructor(props) {
+    super(props)
+    this.model = props.model
+    this.listEditor = new ListEditorView({
+      model: new List(),
+      showListTemplates: true,
+      showFooter: false,
+    })
+  }
+
+  createList = () => {
+    this.listEditor.save()
+    store
+      .getCurrentWorkspace()
+      .get('lists')
+      .add(this.listEditor.model)
+  }
+
+  createListWithBookmarks = () => {
+    this.listEditor.save()
+    if (
+      this.model.every(result => {
+        return result.matchesCql(this.listEditor.model.get('list.cql'))
+      })
+    ) {
+      this.listEditor.model.addBookmarks(
+        this.model.map(function(result) {
+          return result.get('metacard').id
+        })
+      )
+      store
+        .getCurrentWorkspace()
+        .get('lists')
+        .add(this.listEditor.model, { preventSwitch: true })
+    } else {
+      this.listenTo(
+        ConfirmationView.generateConfirmation({
+          prompt:
+            "This list's filter prevents the result from being in the list.  Create list without result?",
+          no: 'Cancel',
+          yes: 'Create',
+        }),
+        'change:choice',
+        function(confirmation) {
+          if (confirmation.get('choice')) {
+            store
+              .getCurrentWorkspace()
+              .get('lists')
+              .add(this.listEditor.model, { preventSwitch: true })
+          }
+        }.bind(this)
+      )
+    }
+  }
+
+  render() {
+    let buttonText, onClickFunction
+
+    if (this.props.withBookmarks) {
+      buttonText = 'Create New List with Result(s)'
+      onClickFunction = this.createList.bind(this)
+    } else {
+      buttonText = 'Create New List'
+      onClickFunction = this.createListWithBookmarks.bind(this)
+    }
+    return (
+      <CreateContainer>
+        <MarionetteRegionContainer view={this.listEditor} />
+        <div className="editor-footer">
```

Comment:
Is this being targeted by something? If not, we can remove the className. If a parent is styling this, we should move that styling into this file and remove the className.